requirements/dev.txt

```diff
@@ -3,5 +3,5 @@
 
 django-debug-toolbar
 sphinx
-django>=1.4,<1.10
```

Comment:
@brad Don't we want to keep these upper limits so that we don't start new versions of requirements before we have a chance to test them?